privacy-security-questionnaire.md

```diff
@@ -0,0 +1,76 @@
+## Self-Review Questionnaire: Security and Privacy
+
+| 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+This feature is a Document-Policy header that an origin may choose to include in an HTTP resonse when it has no linked resources in the response, to indicate to the user agent that it may optimize resources required to conduct a speculative parse of the response. This will benefit the user in terms of reducing compute requirements on the client, and improve performance of the experience.
+
+| 2.2. Do features in your specification expose the minimum amount of information necessary to enable their intended uses?
+
+Yes. The feature only exposes the minimum required information as a boolean header, to satisfy the use case.
+
+| 2.3. How do the features in your specification deal with personal information, personally-identifiable information (PII), or information derived from them?
+Personal information is any data about a user (for example, their home address), or information that could be used to identify a user, such as an alias, email address, or identification number.
+
+The feature does not relate to PII or any derived information thereof.
+
+| 2.4. How do the features in your specification deal with sensitive information?
+Personal information is not the only kind of sensitive information. Many other kinds of information may also be sensitive. What is or isn’t sensitive information can vary from person to person or from place to place. Information that would be harmless if known about one person or group of people could be dangerous if known about another person or group. Information about a person that would be harmless in one country might be used in another country to detain, kidnap, or imprison them.
+
+The feature is unrelated to sensitive information.
+
+| 2.5. Do the features in your specification introduce new state for an origin that persists across browsing sessions?
+
+No. The feature in the specification only applies to a single response and does not persist across browsing session.
+
+| 2.6. Do the features in your specification expose information about the underlying platform to origins?
+
+No. The feature does not expose information about the underlying platform.
+
+| 2.7. Does this specification allow an origin to send data to the underlying platform?
+
+The specification only allows an origin to send a signal to the user agent on how to better handle the current response to achieve higher performance for the user experience.
```

Comment:
I think the answer is just "no", if you look at the definition of "underlying platform" in this question.